include/app/config.hpp

```diff
@@ -111,6 +111,7 @@ class Config : public QObject {
 
     std::shared_ptr<f1x::openauto::autoapp::configuration::Configuration> openauto_config;
 
+	inline QSettings* get_settings() { return &this->ia_config; };
```

Comment:
This clearly bogus but it's a hack to show how we can do settings with much less overhead (see https://github.com/rsjudka/intelligent-auto/pull/36/files#r429593068)

Reply:
Just to be clear, the alternative to giving everyone access to a singleton Config is to give everyone access to a singleton QSettings.